test/Microsoft.AspNetCore.Server.Kestrel.FunctionalTests/TestServer.cs

```diff
@@ -49,7 +49,11 @@ public TestServer(RequestDelegate app, TestServiceContext context, ListenOptions
 
             Context = context;
 
-            _transport = s_transportFactory.Create(listenOptions, new ConnectionHandler<HttpContext>(listenOptions, context, new DummyApplication(app, httpContextFactory)));
+            // Switch this to test on socket transport
+            var transportFactory = CreateLibuvTransportFactory(context);
+            // var transportFactory = CreateSocketTransportFactory(context);
```

Comment:
Are you adding a base class or something to test both transports? If not, remove comment.